tests/test_examples.py

```diff
@@ -270,3 +289,13 @@ def test_horovod(np, example):
     filename, args = example[0], example[1:]
     filename = os.path.join(EXAMPLES_DIR, filename)
     check_call(horovodrun.split() + [sys.executable, filename] + args)
+
+
+@requires_funsor
+@pytest.mark.parametrize('example', FUNSOR_EXAMPLES)
```

Comment:
Will we want to mark this as stage=funsor rather than stage=examples?

Reply:
I did this manually in `.travis.yml`, since it still makes sense to separate example smoke tests from unit tests